cqrs-user-query-service/src/test/java/dev/amir/userquery/it/UserQueryServiceIt.java

```diff
@@ -46,32 +48,60 @@ void getAllUsersTest() throws Exception {
                 }}
         );
 
-        when(userOutputPort.getAll()).thenReturn(mockUsers);
+        when(userOutputPortMock.getAll()).thenReturn(mockUsers);
 
-        mockMvc.perform(MockMvcRequestBuilders.get("/user").contentType(MediaType.APPLICATION_JSON))
+        mockMvc.perform(get("/user").contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.users[0].name").value("kevin"))
                 .andExpect(jsonPath("$.users[1].name").value("john"))
                 .andExpect(jsonPath("$.users[0].id").value("1"))
                 .andExpect(jsonPath("$.users[1].id").value("2"));
 
-        verify(userOutputPort).getAll();
+        verify(userOutputPortMock).getAll();
     }
 
     @Test
-    public void getUsersByIdTest() throws Exception {
+    public void test_GetUsersById() throws Exception {
         User mockUser = new User();
         String expectedUuid = UUID.randomUUID().toString();
         mockUser.setName("kevin");
         mockUser.setId(expectedUuid);
-        when(userOutputPort.getById(any(String.class))).thenReturn(Optional.of(mockUser));
+        when(userOutputPortMock.getById(any(String.class))).thenReturn(Optional.of(mockUser));
 
-        mockMvc.perform(MockMvcRequestBuilders.get("/user/{userId}", expectedUuid)
+        mockMvc.perform(get("/user/{userId}", expectedUuid)
                         .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(jsonPath("$.user.name").value("kevin"))
                 .andExpect(jsonPath("$.user.id").value(expectedUuid))
                 .andExpect(status().isOk());
 
-        verify(userOutputPort).getById(eq(expectedUuid));
+        verify(userOutputPortMock).getById(eq(expectedUuid));
+    }
+
+    @Test
+    public void test_HandleUnknownException() throws Exception {

```

Comment:
this method name should include the function under test, it could be something like:
`test_HandleUnknownExceptionForGetUsersByIdTest`

Reply:
update this method name